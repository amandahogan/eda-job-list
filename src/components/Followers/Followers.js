import React, { PropTypes } from 'react'
import { Image } from 'semantic-ui-react'

class Followers extends React.Component {
  render () {
    return (
      <Image floated='right' size='mini' src={this.props.avatar} />
    )
  }
}

Followers.propTypes = {
  avatar: PropTypes.string.isRequired
}

export default Followers
