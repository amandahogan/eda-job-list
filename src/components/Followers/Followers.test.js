import { shallow } from 'enzyme'
import React from 'react'

import Followers from './Followers'

describe('<Followers />', () => {
  it('matches snapshot', () => {
    expect(
      shallow(<Followers />).getNode()
    ).toMatchSnapshot()
  })
})
