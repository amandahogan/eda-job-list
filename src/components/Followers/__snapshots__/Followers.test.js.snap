exports[`<Followers /> matches snapshot 1`] = `
<Image
  as="img"
  floated="right"
  size="mini"
  src={undefined}
  ui={true} />
`;
